<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F1_F2_BuySell</class>
 <widget class="QDialog" name="F1_F2_BuySell">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>F1 F2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Stockname</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="lineEdit_Lots">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>No. of Lots</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxBuySell">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>28</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Buy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sell</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_Price">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Buy Sell</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pushButtonSubmit">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_Stockname">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxBuySell</tabstop>
  <tabstop>lineEdit_Stockname</tabstop>
  <tabstop>lineEdit_Price</tabstop>
  <tabstop>lineEdit_Lots</tabstop>
  <tabstop>pushButtonSubmit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
